### DISCUSSION TITLE
any way to upload a version bigger than 500mb?

### DISCUSSION THREAD

**Discussion by animatedosprey** [modrinth/.github#188] [Other discussions]
getting an error saying my modpack version is too big.  I see other packs with more mods than mine has, is there any way to include the extra datapacks and resources required for the client?

**Comment by AnOpenSauceDev** [comment]
This shouldn't be a problem at all normally, even with huge modpacks. It sounds like you bundled all your mods into "overrides", which is a bug with a few versions of prism iirc.
